app/frontend/src/pages/Demo/Demo.tsx

```diff
@@ -14,9 +16,55 @@ function Demo() {
   function checkboxOnChange(e: React.ChangeEvent<HTMLInputElement>) {
     console.log(e.target.checked);
   }
-
+  console.log("changes");
   return (
     <Fragment>
+      <HeaderNav
+        logoDesktop={logoHorizontal}
+        logoMobile={logoStacked}
+        menu={[
+          { name: "Hack for LA", link: "/" },
+          { name: "How to Join", link: "/qualifier" },
+          { name: "Projects", link: "/demo" }
+        ]}
+      />
+      <NotificationBar
+        text={
+          <p>
+            This is a limited view. Please
+            <button
```

Comment:
Just trivia, but this should really be an `<a>` since it acts more like a link than a button.